test/BigInt/comparison.js

```diff
@@ -72,6 +72,19 @@ var tests = [
             assert.isFalse(x > y);
         }
     },
+    {
+        name: "Out of 64 bit range",

```

Comment:
Would be nice to have some tests with ridiculously huge numbers so they exercise your dynamic allocation code. You can use strings, the .repeat method, and eval to avoid typing the numbers in directly: just generate a string with a big literal and eval it.

Reply:
will do